lib/repo_tools.pm

```diff
@@ -125,6 +125,21 @@ sub rmt_mirror_repo {
     assert_script_run 'rmt-cli repo list';
 }
 
+sub prepare_Oss_repo {
```

Comment:
would it be ok to use lowercase "oss" here as well as in the repo name? Just to be consistent with the repo URL, the other names and the other methods from this lib.

But the main question is: Why would you do that? Why add the oss repo at all which is an inconsistent combination with the snapshot repo used to trigger tests?

Reply:
Ok, I updated. 

The reason I add this repo is because some applications (eg. libreoffice-base) not installed by default in some cases. And local iso repo doesn't include these rpm. So, in order to test these cases, we have to install from oss. So, I add oss repo temporary to do installation.